### PR TITLE
Fix #117 by treating .NET Framework 4.6 equivalent to .NET Framework 4.5.

### LINKED ISSUE #117: Do your team have any plan to upgrade current version (4.5) into .NET 4.6?
I have worked on a parser for my project in .NET 4.6, I searched around internet and found out Antlr4cs is a good parser for my requirements, but I recognized that Antlr4cs only work on .NET 4.5 target. Do your team have any plan to upgrade to .NET 4.6 target?

### PR DESCRIPTION
**Pull request by fedotovalex**

### CODE REVIEW

**Comment by sharwell** [comment]
Thanks! Looks like the build failure is an incomplete configuration on my part.

File: runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs [hunk 1]
```diff
@@ -301,7 +301,7 @@ public bool Execute()
                     string framework = TargetFrameworkVersion;
                     if (string.IsNullOrEmpty(framework))
                         framework = "v2.0";
-                    if (framework.StartsWith("v4.5."))
+                    else if (framework.StartsWith("v4.5.") || framework.Equals("v4.6"))
```
Comment (sharwell) [inline runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs]: :question: Would it be better to use `StartsWith` for v4.6 as well?

:thought_balloon: Another option is creating a `Version` and using its comparison operator:

```csharp
Version frameworkVersion = new Version(framework.Substring(1));
if (frameworkVersion >= new Version(4, 5))
    framework = "v4.5";
```

Comment (fedotovalex) [inline runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs]: I like the `Version` class approach better. But is it safe to target all future .NET versions? Or does it make sense to recognize 4.5.x and 4.6.x only?

Comment (sharwell) [inline runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs]: It's more likely to be safe than not safe.

Comment (fedotovalex) [inline runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs]: Ok, done.

**Comment by sharwell** [comment]
This change looks good. Can you submit a pull request to [antlr/antlr4](https://github.com/antlr/antlr4) to update the [contributors.txt](https://github.com/antlr/antlr4/blob/master/contributors.txt) file so I can merge this?

**Comment by fedotovalex** [comment]
Done.

**Pull request merged**